src/main/java/com/google/api/gax/grpc/InstantiatingChannelProvider.java

```diff
@@ -58,34 +58,28 @@
  * http header of requests to the service.
  */
 public final class InstantiatingChannelProvider implements ChannelProvider {
-  private static final String DEFAULT_GAX_VERSION = "UNKNOWN";
+  private static final String DEFAULT_VERSION = "UNKNOWN";
 
   private final ExecutorProvider executorProvider;
   private final CredentialsProvider credentialsProvider;
   private final String serviceAddress;
   private final int port;
-  private final String clientLibName;
-  private final String clientLibVersion;
-  private final String serviceGeneratorName;
-  private final String serviceGeneratorVersion;
+  private final String clientName;
+  private final String clientVersion;
 
   private InstantiatingChannelProvider(
       ExecutorProvider executorProvider,
       CredentialsProvider credentialsProvider,
       String serviceAddress,
       int port,
-      String clientLibName,
-      String clientLibVersion,
-      String serviceGeneratorName,
-      String serviceGeneratorVersion) {
```

Comment:
I still don't understand why we aren't retaining both the gapic version and the gccl version when both are relevant. There is a particular scenario I want recorded: Datastore as of now uses the legacy Google API Client stack, thus it would have `gccl` and not `gapic`; but, when we switch it to using a GAPIC client, then I want `gccl` and `gapic` to both appear.